hhs_oauth_server/settings/base.py

```diff
@@ -421,6 +428,11 @@
                 "handlers": ["console"],
                 "level": "INFO",
             },
+            'django': {
+                'handlers': ['console'],
+                'level': 'INFO',
+                'propagate': True,
```

Comment:
I see what you mean, but I don't think we need propagate, it seems to work with or without it, and so for consistency with everything else, I would say we should omit it.
```suggestion
```